setup.py

```diff
@@ -267,6 +268,26 @@ def get_check_description(name, filterfunc):
         print("Checks templates written to %r" % (filename))
 
 
+def parse_long_description(filename):
+    readme_lines = open(
```

Comment:
Can we use with so that we close the file properly, I know its not needed but its a good pattern to follow.